src/numberArray.js

```diff
@@ -0,0 +1,15 @@
+export default function(a, b) {
+  if (!b) b = [];
+  var na = a ? Math.min(b.length, a.length) : 0,
+    c = b.slice();
+  let i;
+  return function(t) {
+    for (i = 0; i < na; ++i) c[i] = a[i] + (b[i] - a[i]) * t;
+    return c;
+  };
+}
+
+export function isNumberArray(x) {
+  if (ArrayBuffer.isView(x)) return !(x instanceof DataView);
+  return Array.isArray(x) && x.every(i => typeof i == "number");
```

Comment:
I still think it’s preferable to start with the simpler type check and only switch to *array*.every if there’s a use case or user request where it makes a noticeable difference.

Reply:
Let's stay on the safe side, then :p